app/assets/stylesheets/reservations/_index.css.scss

```diff
@@ -3,3 +3,7 @@
   border-radius: 5px;
   margin-bottom: 20px;
 }
+
+#reservations-list {
+  height: 10em;
```

Comment:
depends on the level of code review ^_~

1. shallow - cursory-glance feedback for obvious little things
2. read-only - for style/architecture
3. clone-and-manually-test - needed sometimes

Feedback of type 1 from busy people (Adam lol) has helped me more often than you'd think :)